oci-library/oci-hol/java-management-lifecycle-management/introduction/introduction.md

```diff
@@ -30,7 +30,7 @@ _Estimated Time:_ 60 minutes
 
   * You have signed up for an account with Oracle Cloud Infrastructure and have received your sign-in credentials.
   * You are using an Oracle Linux image on your host machine or compute instance for this workshop.
-  * Access to the cloud environment and resources configured in [Workshop 1](../../java-management/workshops/freetier/index.html?lab=set-up-oci-for-jms/set-up-oci-for-jms).
+  * Access to the cloud environment and resources configured in the workshop [Manage Java Runtimes, Applications and Managed Instances Inventory with Java Management Service](https://apexapps.oracle.com/pls/apex/dbpm/r/livelabs/view-workshop?wid=912).
```

Comment:
Livelabs reviewer do not allow dynamic links like this. Instead, he propose we put the actual livelabs production link instead